### ISSUE TITLE
Deprecated?

### ISSUE THREAD

**Issue by im-kulikov** [nvie/gitflow#6452]
https://github.com/Homebrew/homebrew-core/issues/8471

> If you can convince nvie to officially deprecate https://github.com/nvie/gitflow and point to git-flow-avh then we'll consider it. Otherwise, we're going to fall back to the relative upstream names.

**Comment by azeveco** [comment]
Yes. You should use the [Git Flow AVH Edition](https://github.com/petervanderdoes/gitflow-avh).

**Comment by im-kulikov** [comment]
@nvie can you mark this repo as archived and homebrew package as deprecated?

**Comment by wnagrodzki** [comment]
@nvie I would also like to support the idea to mark this repository as no longer maintained. If this is no ideal situation for you, maybe you could mention `git-flow (AVH Edition)` in readme file?

I wish I was aware of AVH Edition. It works with release branches in the same manner Source Tree does. That would save me confusion.

**Comment by petervanderdoes** [comment]
@wnagrodzki Sourcetree has embraced the AVH Edition. It is mentioned in the acknowledgements, About -> Acknowledgements

**Comment by wnagrodzki** [comment]
@petervanderdoes Good to know! I think this information should be advertised prominently in AVH Edition readme file (mention other tools if they us it). That would have made my decision to use it immediate, since the rest of my team uses Source Tree.

**Comment by jawwad** [comment]
Tower also mentions that they use git-flow-avh under the **Installing git-flow** section of [this page](https://www.git-tower.com/learn/git/ebook/en/desktop-gui/advanced-topics/git-flow).

> A couple of different forks of git-flow have emerged in recent years. In this chapter, we're using one of the most popular ones: the AVH Edition.
>
> Using Tower, you don't have to install anything as the app already includes the git-flow scripts.

**Comment by bogdaniel** [comment]
So.. in the end which version should we use .. because I've seen that AVH Edition doesn't seem very active.

**Comment by buhtz** [comment]
https://github.com/nvie/gitflow/issues/6484

**Comment by paulwwisl** [comment]
As of 2023

https://github.com/CJ-Systems/gitflow-cjs is active
https://github.com/petervanderdoes/gitflow-avh is still since May 23, 2019
https://github.com/nvie/gitflow is still since Sep 25, 2012

**Comment by shaedrich** [comment]
[petervanderdoes/gitflow-avh](/petervanderdoes/gitflow-avh) got archived on Jun 19, 2023 without any explanation in the README.

**Comment by nvie** [comment]
Closing all issues as I'm archiving this repo. git-flow lives on at [git-flow-next](https://github.com/gittower/git-flow-next) if you're still using it. Thanks for reporting!
